src/panel/settings/settings_raven_widgets.vala

```diff
@@ -227,7 +227,35 @@ namespace Budgie {
 				return;
 			}
 
-			raven.create_widget_instance(widget_id);
+			var result = raven.create_widget_instance(widget_id);
+			if (result != RavenWidgetCreationResult.SUCCESS) {
+				string markup = null;
```

Comment:
Are these markup strings a user orientated  string and/or developer orientated?  If user orientated then need to surround then strings with _(...) type stuff to make them translatable.

Reply:
I imagine they are developer-oriented images that could help facilitate debugging.

Reply:
Developer-oriented, since a user should never see them if their plugins are set up correctly. They should definitely be translatable, though.